src/app/selectors.ts

```diff
@@ -81,9 +82,10 @@ export const showRaiseDailyLimitSelector = createSelector(
   }
 )
 
+// Added isE2EEnv to test supercharge behavior in E2E tests
```

Comment:
could we move this modification of behaviour to the UI component instead? this feels quite deep in the stack to be making modifications, and is a little opaque/unexpected?

Reply:
to elaborate, this made me think about about [this comment](https://github.com/valora-inc/wallet/pull/1703#discussion_r777616577) from a while ago, about how we can make it more explicit what `isE2EEnv` is doing. I see that for this test, we are trying to mock that supercharge rewards exist, maybe the UI is the least surprising place to put this?

Reply:
Will move to the side menu component.

Reply:
Fixed in https://github.com/valora-inc/wallet/pull/2526/commits/4db5627688b4e42505dd82ea119ed0caf9163de3!